.helmignore

```diff
@@ -0,0 +1,28 @@
+# Patterns to ignore when building packages.
```

Comment:
same feedback as the github workflows at the root of the repo, i'm not sure they are meant to be here. Seems like you need to copy in the directory structure from https://github.com/replicatedhq/platform-examples/pull/1

Reply:
agreed, removed.